src/app/docs/release-notes/rss.xml/route.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable import/prefer-default-export */
+import Rss from 'rss';
+
+import { RELEASE_NOTES_BASE_PATH } from 'constants/docs';
+import { RELEASE_NOTES_DIR_PATH, getAllReleaseNotes, getPostBySlug } from 'utils/api-docs';
+import getReleaseNotesDateFromSlug from 'utils/get-release-notes-date-from-slug';
+
+const SITE_URL = process.env.NEXT_PUBLIC_DEFAULT_SITE_URL;
+
+export async function GET() {
+  const allReleaseNotes = await getAllReleaseNotes();
+
+  const feed = new Rss({
+    id: RELEASE_NOTES_BASE_PATH,
+    language: 'en',
+    title: `Release notes — Neon Docs`,
+    description: 'The latest product updates from Neon',
+    feed_url: `${SITE_URL}${RELEASE_NOTES_BASE_PATH}rss.xml`,
```

Comment:
```suggestion
    feed_url: `${SITE_URL}${RELEASE_NOTES_BASE_PATH}/rss.xml`,
```
?

Reply:
there is a slash here if I'm not mistaken
`const RELEASE_NOTES_BASE_PATH = `${DOCS_BASE_PATH}${RELEASE_NOTES_SLUG}/`;`